lib/task-graph.js

```diff
@@ -522,6 +524,35 @@ function taskGraphFactory(
         }));
     };
 
+    TaskGraph.prototype.updateGraphProgress = function(progressData) {
+        var pattern = /\d{1,3}\.?\d{0,2}\%/;
```

Comment:
Here I want to set restriction on all progress data format to make it look clean. Two digital after point should be enough.